app/controllers/submit.js

```diff
@@ -279,6 +281,27 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
         const modelDOI = this.get('model.doi');
         return (basicsDOI || modelDOI) && basicsDOI !== modelDOI;
     }),
+    basicsLicense: Ember.computed('model', function() {
+        let record = this.get('model.licenseRecord');
+        let license = this.get('model.license');
+        return {
+            year: record ? record.year : null,
+            copyrightHolders: record && record.copyright_holders ? record.copyright_holders.join(',') : null,
+            licenseType: license || null
+        }
+    }),
+    licenseChanged: Ember.computed('model.license', 'model.licenseRecord', 'model.license', function() {
+        let record = this.get('model.licenseRecord');
+        let license = this.get('model.license') || null;
+        let changed = false;
+        if (record) {
+            changed = (record.year != this.get('basicsLicense.year')) || (this.get('basicsLicense.copyrightHolders') !== record.copyrightHolders);
+        } else {
+            changed = (this.get('basicsLicense.year') !== null) || (this.get('basicsLicense.copyrightHolders') || []);
+        }
+        changed = changed || (this.get('basicsLicense.licenseType') !== license);
+        return changed;
+    }),
     basicsChanged: Ember.computed('tagsChanged', 'abstractChanged', 'doiChanged', function() {
```

Comment:
basicsChanged will also need to be watching licenseChanged.  This will determine if the 'Discard Changes' button is enabled, and if the save icon is red or green.